mastodon/setup-mastodon.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ -f ./mastodon.env ] ; then
+    echo "Mastodon already initialised, nothing to do."
+    exit 0
+fi
+
+# Get image versions, admin account
+source ./versions.env
+source ./admin.env
+
+echo "Starting first-time set-up."
+
+temp_cfg=$(mktemp)
+cat ./mastodon.env.template >> "${temp_cfg}"
+
+echo "Generating keys and secrets."
+
+secret="$(docker run -i --network mastodon-net-internal --rm -e RUBYOPT=-W0 \
+            ${MASTODON_IMAGE} bundle exec rake secret)"
+echo "SECRET_KEY_BASE=${secret}" >> "${temp_cfg}"
+
+secret="$(docker run -i --network mastodon-net-internal --rm -e RUBYOPT=-W0 \
+            tootsuite/mastodon:v3.5.3 bundle exec rake secret)"
+echo "OTP_SECRET=${secret}" >> "${temp_cfg}"
+docker run -i --network mastodon-net-internal --rm -e RUBYOPT=-W0 \
+            ${MASTODON_IMAGE} bundle exec rake mastodon:webpush:generate_vapid_key \
+    >> "${temp_cfg}"
+
+chmod 444 "${temp_cfg}"
```

Comment:
Seems like the reason that this file is world readable is that it's being mounted into the docker container below. Could you add a comment about it?

Reply:
😱 That's a left-over from early development; should be *a lot* stricter. I'll update this.

Reply:
Done.